code/modules/hydroponics/seed_machines.dm

```diff
@@ -41,6 +41,15 @@
 	var/failed_task = 0
 	var/disk_needs_genes = 0
 
+
+
+/obj/machinery/botany/can_harvest_biomass()
```

Comment:
/obj/machinery/portable_atmospherics/hydroponics

This is the correct path type. I checked it myself just now.